news/_posts/2020-03-11-improvements-on-contributors-board.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Improvements to come on contributors board"
+subtitle: "More features, more modern, and more information about contributors"
+date:  2020-03-12 15:30:00
+authors: [AntoineThomas]
+icon: icon-board
+image: /assets/images/theme/meta-logo-build.png
+tags: [contribute, community]
+---
+
+
+After all those years without any modification nor improvements, it is really time to think about the future of the "[Top contribution achievements](http://contributors.prestashop.com/)" board. While discussing with many people from the community on how to open the governance of the open source project, and how to involve more contributors, new needs have been identified. This article is about quick wins, upcoming developments, and about the general spirit behind this work.
+
+## But, why?
+
+In 2020, a lot of work is going to be done around the PrestaShop open source project to make it more attractive for contributors. It must of course take into account that most people don't really contribute out of generosity, because at the end of the month, everybody has to pay their bills. So, like with other well-known open source projects, people and companies who take time to contribute to PrestaShop must be able to show off: being visible in the community as an expert is valuable to attract customers, it provides proof that they know what they are doing with their PrestaShop business.
+
+There are many ideas that are currently being tested, or will be soon:
+
+- [Call for papers at PrestaShop events](https://build.prestashop.com/news/CFP-PrestaShop-Day-Paris-2020/), giving a place for community speakers
+- Opening the maintainer role on a GitHub repository with selected developers, in order to identify problems and document how to do it well;
+- Opening a chat platform for contributors (due date on Q1)
+- ... and this is just the tip of the iceberg
+
+But, at the end, the question remains: how can a contributor show their knowledge, their expertise, and their investment in the project to their customers.
+
+This is why it is time to improve the Contributors' board. Let's see what is on the schedule already.
+
+
+## The current plan
+
+### First step: quick wins
+
+If you go on [contributors.prestashop.com](http://contributors.prestashop.com/), it is easy to notice a few issues:
+
+- Some contributors like PrestaShop employees are not listed, although they do a huge job on the open source project since the very beginning, and this work is currently hidden
+- It is not possible to see the entire list of contributors, without clicking on "see more", or on page numbers
+- The board lists only contributions to the core; but the PrestaShop software is also made of modules, and of course, there are other very important contributions like specifications, documentation, blog articles, and many more. 
+
+So, it became obvious that the first step must be to show the work of everyone involved in the making of the PrestaShop open source software project.
```

Comment:
Should we keep 'software' here?

Reply:
at the end, the project is delivered as a software.